src/CatalogPage.js

```diff
@@ -1,10 +1,53 @@
+import { it } from "node:test";
+
 /**
  * This class obtains the elements in the DOM for the
  * catalog page using different locators
  */
 class CatalogPage {
   constructor(page) {
     this.productTitle = page.getByTestId("title");
+    this.productPrices = page.getByTestId("inventory-item-price");
+    this.dropdown = page.locator("//select[@data-test='product-sort-container']");
   }
+
+  async getPriceListArray(){
+    const priceItemsArray = [];
+    for(let item of await this.productPrices.all()){
+      const price  = await item.innerText();
+      const priceFloat = parseFloat(price.replace("$", ""));
+      priceItemsArray.push(priceFloat);
+    }
+    return priceItemsArray;
+  }
+
+  async orderArrayAscending(itemsPriceArray){
+    var pricceItemOrdered = itemsPriceArray.sort(function (a,b) {
+      return a - b; // Ascending
+  });;
+  return pricceItemOrdered;
+  }
+
+  async selectOrderBy(){
```

Comment:
`selectOrderBy` could receive as an argument a string where you pass the option intended to be selected.
![Screenshot 2024-10-05 at 8 11 40 a m](https://github.com/user-attachments/assets/e8f34058-a21b-42c4-9363-1c42765b6df7)